src/main/java/racing/domain/RacingGame.java

```diff
@@ -0,0 +1,23 @@
+package racing.domain;
+
+import racing.ui.ResultView;
+
+public class RacingGame {
+
+  private final RoundCount tryCnt;
+  private final Game game;
+
+  public RacingGame(Game game, RoundCount tryCnt) {
+    this.game = game;
+    this.tryCnt = tryCnt;
+  }
+
+  public void play() {
+    ResultView.printResultMessage();
+    for (int i = 0; i < tryCnt.getCount(); i++) {
```

Comment:
```suggestion
    for (tryCnt.racing()) {
```
위와 같이 메시지를 보내보면 어떨까?
즉, play를 할 때마다 tryCnt의 값을 1씩 감소시켜 0이면 race가 끝...
객체 지향적인 사고 연습해 보면 어떨까?